addon/utils/data-table.js

```diff
@@ -0,0 +1,50 @@
+export function measureColumn(tableData) {
```

Comment:
Some slight difference to the original here
![Screen Shot 2019-04-04 at 9 01 27 PM](https://user-images.githubusercontent.com/160955/55577783-07eab080-571d-11e9-9cdd-6f97d0376cd0.png)